Dockerfile

```diff
@@ -1,14 +1,23 @@
-FROM nodesource/node:trusty
+FROM nodesource/trusty:4.4.7
 MAINTAINER Ben Kero <ben.kero@gmail.com>
 
-ADD package.json /tmp/package.json
 RUN apt-get update && apt-get install -y git-core
-RUN cd /tmp && npm install
-RUN mkdir -p /hyperchannel && cp -a /tmp/node_modules /hyperchannel/
+
+# The Docker image now sets NODE_ENV="production". All our dependencies are
+# devDependencies
+ENV NODE_ENV development
```

Comment:
I would suggest putting this near the command that this environment variable affects